src/isSet.js

```diff
@@ -0,0 +1,25 @@
+import { is } from 'ramda';
+import curry1 from 'ramda/src/internal/_curry1';
+
+/**
+ * Predicate for determining if a provided value is an instance of a Set.
+ *
+ * @func isSet
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.22.0|v2.22.0}
+ * @category Type
+ * @sig * -> Boolean
+ * @param {*} val The value to test
+ * @return {boolean}
+ * @see {@link RA.isSet}
+ * @example
+ *
+ * RA.isSet(new Map()); //=> false
+ * RA.isSet(new Set()); //=> true
+ * RA.isSet(new Set([1,2]); //=> true
+ * RA.isSet(new Object()); //=> false
+ */
+
+const isSet = curry1(val => is(Set, val));
```

Comment:
We're not using `R.is` in this library anymore. It's has a lot of problems. One of the main problem is described here: https://github.com/char0n/ramda-adjunct/issues/724

instead we prefer to use [tagChecking](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toStringTag) which solves the problem. 

Pls aligned your implementation with `isDate` function - https://github.com/char0n/ramda-adjunct/blob/96d0772005ddf262ef2871ca5484dc156eb49805/src/isDate.js#L21

Reply:
Moved over to the pipe, type, identical standard